src/pages/[platform]/build-a-backend/auth/add-social-provider/index.mdx

```diff
@@ -946,6 +946,20 @@ function App() {
 
 </InlineFilter>
 
+### Custom Providers
+
+When using custom providers that are not provided by default in Cognito, you can pass an object to the `provider` parameter with the name of your custom provider.
+
+```javascript
+import { signInWithRedirect } from 'aws-amplify/auth';
+
+const provider = {
+  custom: 'MyCustomOIDCProvider'
+}
+
+<button onClick={() => signInWithRedirect({ provider })}> MyCustomOIDCProvider</button>
```

Comment:
This is a React specific example, won't this be weird on the Vue/Angular pages? Feels like we should try to stick to framework agnostic examples or have examples for each framework. What do you think of showing a handleButtonClick function instead of showing the button element and including a comment saying they should attach it to their button?